### ISSUE TITLE
term/session/meeting/sitting organization

### ISSUE THREAD

**Issue by matyaskopp** [ufal/ParlaMint-UA#12]
Currently, I am able to categorize one file with:
- term
- session
- sitting (day + number of sitting in a day )

Source for this categorization is https://data.rada.gov.ua/open/data/stenogram_ses-skl8

https://github.com/ufal/ParlaMint-UA/blob/2c7cf1e3121083b2074bbc6cd060003dff72611c/SampleData/03-tei-text-lang/ParlaMint-UA_2015-02-13-m0.xml#L7-L9

@AnnaParla  Is there any source of `meeting` number?

I don't understand what this mean:
https://github.com/ufal/ParlaMint-UA/blob/2c7cf1e3121083b2074bbc6cd060003dff72611c/SampleData/03-tei-text-lang/ParlaMint-UA_2015-02-13-m0.xml#L46
Is that the `meeting` number? Is it possible to get it somewhere in numeric form?

**Comment by AnnaParla** [comment]
There are no numerical meeting numbers in the records which you used but you can find them on the website in the publications heading block, if it helps:
https://www.rada.gov.ua/meeting/stenogr/show/5799.html

<div class="publications-heading__block">
            <h2>Засідання 11
            </h2>
            <p>
                
                Опубліковано 13.&nbsp;02.&nbsp;2015
            </p>

        </div>

**Comment by AnnaParla** [comment]
AlsoAlso, there are zip files with lists of urls for all the meetings in a session. E.g. https://data.rada.gov.ua/open/data/stenogram_ses-skl8
Folders look like this:
![image](https://user-images.githubusercontent.com/115141649/214286964-c96b92eb-4273-450e-a6bf-199fd153142e.png)
And lists look like this, with the last column being the meeting number, but sometimes numbers on the lists are missing (and not only if a meeting is ceremonial).
![image](https://user-images.githubusercontent.com/115141649/214288152-058dbaa2-1dfd-4f95-9ba3-e1941e870486.png)

**Comment by matyaskopp** [comment]
> There are no numerical meeting numbers in the records which you used but you can find them on the website in the publications heading block, if it helps: https://www.rada.gov.ua/meeting/stenogr/show/5799.html
> 
> ## Засідання 11
> ```
>             Опубліковано 13.&nbsp;02.&nbsp;2015
>         </p>
> 
>     </div>
> ```

Yes, but I use a different source, and the transcript starts with
![image](https://user-images.githubusercontent.com/5867995/214344255-1c8f8dfc-4bd5-4de6-81c9-7dbca59889cb.png)
I do not have time to process the text value of the meeting number (there can appear various issues) - postponing this I first want to have a full working pipeline.



> Also, there are zip files with lists of urls for all the meetings in a session. E.g. https://data.rada.gov.ua/open/data/stenogram_ses-skl8

This is what I use

**Comment by AnnaParla** [comment]
Ok, so there are meeting numbers next to urls in these files but some numbers are missing.  
![image](https://user-images.githubusercontent.com/115141649/214538164-4e04a096-34df-498e-a0e7-b2da7d6186cf.png)
If you want me to do anything manually with this issue, let me know.
